### DISCUSSION TITLE
希望实现 CTRL+Shift+C组合键复制文件路径

### DISCUSSION THREAD

**Discussion by linuxlionking** [linuxdeepin/developer-center#3445] [Features Request & Ideas | 特性请求 & 头脑风暴]
如题。Windows 11 已支持。用户能够使用 CTRL+ Shift + C 的组合键，拷贝选定文件的存储路径。
若能实现，可方便不少。

**Comment by Decodetalkers** [comment]
> 如题。Windows 11 已支持。用户将能够使用 CTRL+ Shift + C 的组合键，拷贝选定文件的存储路径。 若能实现，可方便不少。

你ctrl c下，然后复制到文本编辑器里看看？这是已经有的功能啊

**Reply by linuxlionking** [reply]
我若是想把文件路径发送给微信好友呢？

**Reply by BLumia** [reply]
先贴在只支持纯文本的地方中转一下吧。目前复制到剪切板里的内容，粘贴的时候会贴到什么是取决于你所粘贴到的应用的，剪切板里的内容实际上是给接受不同类型（MimeType）内容的应用设置了对应的内容的。

**Comment by BLumia** [comment]
> Windows 11 已支持。用户将能够使用 CTRL+ Shift + C 的组合键，拷贝选定文件的存储路径。

我在 Windows 11 21H2 测试了一下，在文件资源管理器使用 Ctrl + Shift + C 组合键并不能复制得到一个文件路径呀。是不是你有装什么第三方组件座了做了这件事？

**Reply by linuxlionking** [reply]
我的是win 11 22H2 (22622.601) 版本，系统原生支持的。只是提出一下期望，如果能实现更好。

**Comment by 1-erick-zhang** [comment]
我的建议也是添加，中转一下麻不麻烦，而且win11右键也能复制

**Reply by BLumia** [reply]
Windows 11 21H2 右键也没有单独的复制路径的选项呀，你也是 22H2 吗？可以提供个右键菜单里关于复制路径的选项的截图吗？（想看看是怎么考虑文案以及是放在哪里的）

**Reply by linuxlionking** [reply]
![1111](https://user-images.githubusercontent.com/7598566/192927990-253c1780-fa1e-417e-810b-0a957ada8e9b.png)
如图

**Reply by BLumia** [reply]
cc @meiyixiang
